src/linter_config.yaml

```diff
@@ -0,0 +1,8 @@
+extends: recommended
```

Comment:
We have all our linter configs in the root of the repo. Maybe it would be better to move it there and rename it to something like ".linkml_linter.yaml".